packages/@webex/internal-plugin-mercury/src/mercury.js

```diff
@@ -399,8 +401,11 @@ const Mercury = WebexPlugin.extend({
   },
 
   _getEventHandlers(eventType) {
-    const [namespace, name] = eventType.split('.');
     const handlers = [];
+    if (!eventType) {
```

Comment:
Why did we need this change?

Reply:
this is  just null/undefined check because this is designed for mercury but cc webscoket handlers not required.

Reply:
This new condition doesn't have a UT..